packages/mc-scripts/package.json

```diff
@@ -59,7 +59,7 @@
     "postcss": "7.0.17",
     "postcss-color-mod-function": "3.0.3",
     "postcss-custom-media": "7.0.8",
-    "postcss-custom-properties": "8.0.10",
+    "postcss-custom-properties": "9.0.1",
```

Comment:
requires node 8 or higher

Reply:
hm, this is breaking our builds, resulting in "unknown word" errors

Reply:
[ref](https://github.com/postcss/postcss-custom-properties/issues/186)

Reply:
Wät.